### DISCUSSION TITLE
Auto insert mode in commit message box

### DISCUSSION THREAD

**Discussion by Steellow** [JetBrains/ideavim#651] [Q&A]
Is it possible to disable IdeaVim in the commit message box? Or automatically go to insert mode? I don't really need any Vim stuff when typing commit message, and also it's one extra step to always go into insert mode.

**Comment by ljchen1** [comment]
Hi,

Do you use the Commit dialog instead of the Commit tool window? AFAIK, the IDEA default to insert mode in the Commit tool window.

If you are using the Commit dialog, try adding `set ideavimsupport=` to disable the IdeaVIM support in the dialog.
See https://github.com/JetBrains/ideavim/wiki/set-commands
